META-INF/plugin.xml

```diff
@@ -113,6 +113,10 @@
             text="Restart ghc-modi" description="Restarts a disabled ghc-modi process">
       <add-to-group group-id="ToolsMenu" anchor="last"/>
     </action>
+      <action id="TypeInfo" class="com.haskforce.highlighting.annotation.external.TypeInfoAction" text="Type Info">
+          <add-to-group group-id="ViewMenu" anchor="first"/>
+          <keyboard-shortcut first-keystroke="alt EQUALS" keymap="$default"/>
+      </action>
```

Comment:
Could you update the registration keybindings to include `F11` for Mac?  The `⌥ =` combination doesn't work well on Mac without tampering with the global keyboard configuration.  Also (nit-pick), could you indent the XML at the same level as the other configurations?
```xml
<action id="TypeInfo" class="com.haskforce.highlighting.annotation.external.TypeInfoAction" text="Type Info">
  <add-to-group group-id="ViewMenu" anchor="first"/>
  <keyboard-shortcut first-keystroke="F11" keymap="Mac OS X"/>
  <keyboard-shortcut first-keystroke="F11" keymap="Mac OS X 10.5+"/>
  <keyboard-shortcut first-keystroke="alt EQUALS" keymap="$default"/>
</action>
```